CITATION.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+title: pyrealm
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: C. David L.
```

Comment:
Do you want to list further contributors? Alex, Tian, Surbhi, James, Amy, Vivienne, Marion?

Reply:
Yes -this is totally out of sync with the authors in `pyproject.toml`.  Had forgotten about it.